WordFrequency.java

```diff
@@ -0,0 +1,4 @@
+public interface Wordfrequency {
```

Comment:
In its current state, the code will not compile. Please rename the interface to `WordFrequency` with a capital "F".
A public interface's name should be exactly the same as the filename it is in.

Reply:
Updated. Thanks!